packages/@webex/plugin-cc/test/unit/spec/services/AgentConfigService.ts

```diff
@@ -0,0 +1,200 @@
+import {WebexSDK} from '../../../../src/types';
```

Comment:
Why are these files appearing in your changes? Shouldn't this just show me rename change? Please ensure that we are not logging changes from Parv's PR

Reply:
i changed the folder structure and refactored few things. so, they are appearing.